lte/gateway/python/integ_tests/s1aptests/test_attach_and_mme_restart_loop_detach_and_mme_restart_loop_multi_ue.py

```diff
@@ -14,6 +14,7 @@
 import random
 import time
 import unittest
+from datetime import datetime
```

Comment:
Since we are still imprting `time` library, instead of importing one more library `datetime`, can we use something from `time` library like `time.time()` or `time.perf_counter()` or `time.process_time()`

Reply:
Good point, I now use `time.time()` instead.